test/unit/lib/plugins/aws/package/compile/events/httpApi.test.js

```diff
@@ -125,6 +125,32 @@ describe('lib/plugins/aws/package/compile/events/httpApi.test.js', () => {
       const apiResource = cfResources[naming.getHttpApiLogicalId()];
       expect(apiResource.Properties.DisableExecuteApiEndpoint).to.equal(undefined);
     });
+
+    it('should support payload format version per function', async () => {
+      const { awsNaming, cfTemplate } = await runServerless({
```

Comment:
We could avoid extra `runServerless` run if you could reuse the `runServerles` result that are setup in `before` step. Could you adjust configuration that is run in before so it will also support testing this case? Please see how tests above are configured